### PR TITLE
Enhancement Community Member Profile Social Links 

### LINKED ISSUE #4367: Enhancement Community Member Profile Social Links
### Desired Behavior

Add Meshery Cloud public profile link of the member next to LinkedIn icon.

1. Add Meshery Cloud logo next to the LinkedIn icon.
2. Add meshery cloud UUID as input in "member-profile-template".

https://layer5.io/community/members/aadhitya-amarendiran

### Screenshots / Mockups

Screenshot 2023-06-09 at 2 32 30 PM

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Ghat0tkach**
**Description**

This PR fixes #4367 

**Notes for Reviewers**
-  Added meshery cloud profile and badge
- All you need is to enter meshery-uuid in templates!!
- Also added mesherycloud__uuid for @Freedisch to test the changes
 [Live Preview](https://64a2fe05ac33f734aafd49db--layer5.netlify.app/community/members/freedisch-thibaut-batale)

image

## Result
image

A PR was already drafted in #4402 , however it isnt functional .
**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 77809bffd8d6eaaf7c5c3734f7e791640c9d65ba at: https://64a2fe05ac33f734aafd49db--layer5.netlify.app

**Review by 8rxn — APPROVED** [review]
Great work on the PR @Ghat0tkach 

### LGTM 🚀

File: gatsby-node.js [hunk 1]
```diff
@@ -718,6 +718,7 @@ exports.createSchemaCustomization = ({ actions }) => {
        eurl: String,
        twitter: String,
        github: String,
+       mesherycloud_uuid: String,
```
Comment (leecalcote) [inline gatsby-node.js:721]:
```suggestion
       layer5: String,
```

File: src/collections/members/_member-profile-template/index.mdx [hunk 2]
```diff
@@ -5,6 +5,7 @@ image_path: ./display-picture.webp
 github: # just the user handle (e.g. vinayaksh42)
 twitter: # just the user handle (e.g. Vinayak47427793)
 linkedin: # last portion of https://www.linkedin.com/in/<profile-name> (e.g. vinayak-sharma-141096193)
+mesherycloud_uuid: # last portion of https://meshery.layer5.io/user/<meshery-uuid>
```
Comment (leecalcote) [inline src/collections/members/_member-profile-template/index.mdx:8]:
```suggestion
layer5: # Layer5 Cloud user ID; last portion of https://meshery.layer5.io/user/<uuid>
```

File: src/collections/members/freedisch-thibaut-batale/index.mdx [hunk 3]
```diff
@@ -5,6 +5,7 @@ image_path: ./freedisch.webp
 github: Freedisch
 twitter: freedisch
 linkedin: thibaut-freedisch-batale-905843208
+mesherycloud_uuid: a551897f-da39-4d4d-812e-6b299e82c457
```
Comment (leecalcote) [inline src/collections/members/freedisch-thibaut-batale/index.mdx:8]:
```suggestion
layer5: a551897f-da39-4d4d-812e-6b299e82c457
```

File: src/templates/member-single.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ export const query = graphql`query MemberBySlug($slug: String!) {
       position
       github
       twitter
+      mesherycloud_uuid
```
Comment (leecalcote) [inline src/templates/member-single.js:15]:
```suggestion
      layer5
```

**Review by leecalcote — APPROVED** [review]
LGTM. Good work, @Ghat0tkach 👍

**Comment by l5io** [comment]
🚀 Preview for commit e4492914694420eabd2876a84898ba66acf256ae at: https://64a333bef8f8e65815ff4840--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
Thank you @leecalcote and @8rxn .

**Comment by leecalcote** [comment]
@Ghat0tkach what's the plan for propagating these UUIDs?

**Comment by Ghat0tkach** [comment]
> @Ghat0tkach what's the plan for propagating these UUIDs to other profiles?

After your recent commit , I need to do only one change to make it work again.
It will be like any other social id. We just need to enter the user's meshery-uuid and it will redirect to the meshery cloud user profile itself.

**Comment by l5io** [comment]
🚀 Preview for commit a81201b80814d77c8116adedc7867359afcdf2fd at: https://64a33c0a0e351368df3f92aa--layer5.netlify.app

**Comment by leecalcote** [comment]
> @leecalcote
> Nothing to worry , It will work same as other social ids [linkedin,github and twitter] . All we need to add is `layer5:{user-id}` in member's profile.

Yes, I know. What I was suggesting in a roundabout way is that you go add UUIDs to the rest of the profiles.

**Comment by Ghat0tkach** [comment]
> > @leecalcote
> > Nothing to worry , It will work same as other social ids [linkedin,github and twitter] . All we need to add is `layer5:{user-id}` in member's profile.
> 
> Yes, I know. What I was suggesting in a roundabout way is that you go add UUIDs to the rest of the profiles.

Will Do , Definately 🙌

**Comment by leecalcote** [comment]
Thank you. 😀

**Comment by leecalcote** [comment]
@Ghat0tkach will you open a new issue to track the update of the rest of the profiles?

**Comment by Ghat0tkach** [comment]
> @Ghat0tkach will you open a new issue to track the update of the rest of the profiles?

Yes sure , on my way 🚀

**Pull request merged**